control/vehicle_cmd_gate/config/vehicle_cmd_gate.param.yaml

```diff
@@ -6,8 +6,15 @@
     stop_hold_acceleration: -1.5
     emergency_acceleration: -2.4
     stopped_state_entry_duration_time: 0.1
-    vel_lim: 25.0
-    lon_acc_lim: 5.0
-    lon_jerk_lim: 5.0
-    lat_acc_lim: 5.0
-    lat_jerk_lim: 5.0
+    nominal:
```

Comment:
When is it used this param?

Reply:
nominal: used at any time as before except in TRANSITION mode.
on_transition: used only in TRANSITION mode.